.gitignore

```diff
@@ -15,6 +15,8 @@ vendor/
 
 #Ignore local directory (for stuff to keep locally)
 local/
+!config/deploy/local
```

Comment:
Is this definitely needed? Is it to negate the above `local/`? Seems that git will pick up those files even without these entries.

Reply:
If you try and add a directory anywhere called "local" the `local/` in the gitignore prevents you from ever being able to add it. Seems like it doesn't care if it's already committed, so `config/deploy/local` was fine, i just added it for consistency.

Reply:
👍 i only tried with editing.